dashboard/db/migrate/20170509020232_create_pd_workshop_survey.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePdWorkshopSurvey < ActiveRecord::Migration[5.0]
+  def change
+    create_table :pd_workshop_surveys do |t|
+      t.references :pd_enrollment, null: false, index: true, unique: true
```

Comment:
Upon further investigation, it looks like all you actually have to do is is pass `{unique: true}` as the argument to `index`! TIL